src/main/webapp/app/entities/ace-editor/annotation.model.ts

```diff
@@ -1 +1 @@
-export type AceAnnotation = { row: string; column: string; text: string; type: string };
+export type AceAnnotation = { row: number; column: number; text: string; type: string; ts: number };
```

Comment:
would it make sense to use a proper class type here?

Reply:
Actually thinking about it, it would make sense to move the updateAnnotation from the component to a proper Annotation class for reuse purposes. Good idea!

Reply:
Done.